db/change_cache_test.go

```diff
@@ -443,102 +436,31 @@ func TestLateSequenceHandlingDuringCompact(t *testing.T) {
 
 	// Wake everyone up to detect termination
 	// TODO: why is this not automatic
-	WriteDirect(t, db, channelSet, 4)
+	WriteDirect(t, collection, channelSet, 4)
 	changesFeedsWg.Wait()
 
 }
 
-// Create a document directly to the bucket with specific _sync metadata - used for
-// simulating out-of-order arrivals on the tap feed using walrus.
-
-func WriteDirect(t *testing.T, db *Database, channelArray []string, sequence uint64) {
-	docId := fmt.Sprintf("doc-%v", sequence)
-	WriteDirectWithKey(t, db, docId, channelArray, sequence)
-}
-
-func WriteUserDirect(t *testing.T, db *Database, username string, sequence uint64) {
+func writeUserDirect(t *testing.T, db *Database, username string, sequence uint64) {
```

Comment:
I think it would make sense to update this function to use the metadata store (i.e. db.MetadataStore.Add(....)) while you're cleaning these up.